resources/user-agents/browsers/chrome/chrome-ios-48-to-64.json

```diff
@@ -24,6 +24,7 @@
             "iPod": "iPod Touch"
           },
           "platforms": [
+            "iOS_A_12_4", "iOS_A_12_3", "iOS_A_12_2", "iOS_A_12_1", "iOS_A_12_0",
```

Comment:
Are there tests for these platforms on these versions of Chrome?  I ask because Chrome 64 (the highest version in this file) was released on January of 2018 (https://en.wikipedia.org/wiki/Google_Chrome_version_history) and iOS 12.0 was released in September of 2018 (to the public, it was in developer beta in June of that year https://en.wikipedia.org/wiki/IOS_12).  When the developer beta of iOS 12.0 was released, Chrome was at version 67.

Of course, it's not impossible (I don't believe) for an iOS 12.0->12.4 user to have an older version of Chrome installed (if they have installed the older version before upgrading to iOS 12, and then never updated Chrome), just not sure it's worth adding additional patterns.

I did just scan our logs for Chrome iOS versions, and Chrome < 65 only makes up **0.6%** of all Chrome on iOS 12.x since January of 2018 (when Chrome 64 was released).

Up to you if you think they're worth keeping (only adds 85 patterns).

Reply:
See the test cases `issue-2101-R` and `issue-2101-T`.

Reply:
@jaydiablo I looked on the [Chrome Compatibility](https://en.wikipedia.org/wiki/Google_Chrome#Compatibility) list. The x86 support for Chrome on Linux was removed with Version 49. Shall remove these pattern for Version 50+? Do you have any information about the support for BSD Systems?

Reply:
AFAIK Chrome isn't available at all on BSD systems (officially), but Chromium is.